macros/black_box.config.mac

```diff
@@ -3,14 +3,18 @@
 ### board in the black box.
 ### -----------------------------------------------------------
 
-/Generator/LambertianGenerator/particle opticalphoton
+#/Generator/LambertianGenerator/particle opticalphoton
```

Comment:
I would leave `opticalphoton` as a particle in the example, since it is how we use this macro, most of the times.

Reply:
Done!